src/sinks/loki/tests.rs

```diff
@@ -146,3 +146,32 @@ async fn healthcheck_grafana_cloud() {
         .await
         .expect("healthcheck failed");
 }
+
+#[tokio::test]
+async fn timestamp_out_of_range() {
+    use vector_lib::config::log_schema;
```

Comment:
I meant the beggining of this module, about line 10. It is more idiomatic to have all the use statements at the begginning of the module, so we don't have to dive into each scope and find those statements. 

What do you think?

Reply:
Get it, check the new commit.

Reply:
yes, thats it! Thanks :)